rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/vulnerability/VulnerabilityController.java

```diff
@@ -359,4 +365,40 @@ private Set<Release> getReleaseRelationsInfo(Vulnerability vulnerability, User u
 
         return releaseList;
     }
+
+    @GetMapping(value = VULNERABILITIES_URL + "/trackingStatus" + "/{projectId}")
+    public void getVulnerabilitiesTrackingStatus(@PathVariable("projectId") String projectId,
+            HttpServletResponse response) throws TException, IOException {
+        List<JsonObject> resultJson = new ArrayList<>();
+        try {
+            User user = restControllerHelper.getSw360UserFromAuthentication();
+            List<ReleaseClearingStatusData> releaseClearingStatusList = vulnerabilityService
+                    .getReleasesClearingStatusWithAccessibility(user, projectId);
+            releaseClearingStatusList.forEach(rel -> {
+                if (rel.getRelease() != null) {
+                    JsonObject jobj = new JsonObject();
+                    jobj.addProperty("releaseId", rel.getRelease().getId());
+                    jobj.addProperty("name", rel.getRelease().getName());
+                    jobj.addProperty("projectOrigin", rel.getProjectNames());
+                    jobj.addProperty("svmTrackingStatus",
+                            null != rel.getRelease().getExternalIds() && !rel.getRelease().getExternalIds().equals("")
```

Comment:
InternalIds  and AddditionalData is a Map so it is always not equal to "". Use isEmpty()